inc/admin/ui/meta-boxes.php

```diff
@@ -69,14 +69,15 @@ function rocket_display_cache_options_meta_boxes() {
 			<p><?php esc_html_e( 'Activate these options on this post:', 'rocket' ); ?></p>
 			<?php
 			$fields = [
-				'lazyload'         => __( 'LazyLoad for images', 'rocket' ),
-				'lazyload_iframes' => __( 'LazyLoad for iframes/videos', 'rocket' ),
-				'minify_css'       => __( 'Minify/combine CSS', 'rocket' ),
-				'minify_js'        => __( 'Minify/combine JS', 'rocket' ),
-				'cdn'              => __( 'CDN', 'rocket' ),
-				'async_css'        => __( 'Optimize CSS Delivery', 'rocket' ),
-				'defer_all_js'     => __( 'Defer JS', 'rocket' ),
-				'delay_js'         => __( 'Delay JavaScript execution', 'rocket' ),
+				'lazyload'          => __( 'LazyLoad for images', 'rocket' ),
+				'lazyload_iframes'  => __( 'LazyLoad for iframes/videos', 'rocket' ),
+				'minify_css'        => __( 'Minify/combine CSS', 'rocket' ),
+				'minify_js'         => __( 'Minify/combine JS', 'rocket' ),
+				'cdn'               => __( 'CDN', 'rocket' ),
+				'async_css'         => __( 'Optimize CSS Delivery', 'rocket' ),
+				'defer_all_js'      => __( 'Defer JS', 'rocket' ),
+				'delay_js'          => __( 'Delay JavaScript execution', 'rocket' ),
+				'remove_unused_css' => __( 'Remove Unused CSS', 'rocket' ),
```

Comment:
I'd suggest moving the Remove Unused CSS option right after the minify CSS option, to preserve the context